### DISCUSSION TITLE
Append relative time not working?

### DISCUSSION THREAD

**Discussion by questskyzeidler** [AprilSylph/XKit-Rewritten#1353] [Help Desk]
Every other element of TimeFormat is working fine, but the relative time has stopped appearing since last time I was on Tumblr two days ago.

**Comment by marcustyphoon** [comment]
Are you on Firefox, and is this fixed if you open a new Tumblr tab, and if not that than if you restart your browser? If so, that lines up with a report from another user I got, and indicates that we might need to do something special to get around Firefox's unusual way of silently replacing extensions with their new versions.

**Reply by questskyzeidler** [reply]
I am on Firefox, I hadn't tried opening it in a new tab and that seems to have fixed it. Oops, I'm glad it was that simple. Thanks for the help!

**Reply by marcustyphoon** [reply]
Awesome, thanks for following up!
